index.js

```diff
@@ -1,17 +1,35 @@
-const fs = require('fs');
-const path = require('path');
+const es5TaskFn = require('./lib/webpack.es5.js');
+const { stripIndent } = require('common-tags');
 
 module.exports = {
-  bundle: {
-    directory: 'modules',
-    modules: getBundleModuleNames()
+  improve: '@apostrophecms/asset',
+  init(self) {
+    self.es5TaskFn = es5TaskFn;
+  },
+  extendMethods(self) {
+    return {
+      configureBuilds(_super) {
+        _super();
+        self.builds['src-es5'] = {
+          // An alternative build from the same sources for IE11
+          source: 'src',
+          webpack: true,
+          scenes: [ 'public', 'apos' ],
+          // The CSS from the src build is identical, do not duplicate it
+          outputs: [ 'js' ],
+          label: 'apostrophe:ie11Build',
+          // Load index.js and index.scss from each module
+          index: true,
+          // The polyfills babel will be expecting
+          prologue: stripIndent`
+            import "${require.resolve('core-js/stable')}";
```

Comment:
The `require.resolve` trick allows us to import these relative to this module's dependencies, and will also find them higher up if they were hoisted.